models/staging/staging_tenant.sql

```diff
@@ -0,0 +1,14 @@
+select
+    tenant_type
+    , honor_declaration
+    , clarification
```

Comment:
On peut se permettre ici d'ajouter un commentaire par ex pour expliquer la signification de `clarification`
```suggestion
    , clarification -- un commentaire pour expliquer le sens de cette colonne mystérieuse ;)
```

Reply:
C'est commenté dans la prochaine version